pkg/cassandra/config/config.go

```diff
@@ -32,6 +32,7 @@ type Configuration struct {
 	LocalDC              string        `yaml:"local_dc"`
 	ConnectionsPerHost   int           `validate:"min=1" yaml:"connections_per_host"`
 	Timeout              time.Duration `validate:"min=500"`
+	ConnectTimeout       time.Duration `validate:"min=600"`
```

Comment:
this shouldn't have `validate` tag, since it's a new optional param, but should have `yaml:"connect_timeout"`

Reply:
Fair point, this is actually optional. Pushing Changes